devel/external/AEGIS/document/main.tex

```diff
@@ -0,0 +1,516 @@
+\item {\tt average\_mask\_adjacent\_pixel\_count}: the average flux in the $3 \times 3$ pixel region around this closest replaced pixel, before replacing with noise.
+\end{enumerate}
+The parameters are computed separately in each band. 
+
+A ``bad stamp'' flag is set if
+\begin{enumerate}
+\item {\tt HSM} module of {\tt GalSim} was not able to compute moments and the SExtractor parameter ELLIPTICITY $\leq $ 0.75, in any filter.
+\item {\tt min\_mask\_dist\_pixels} $\leq $ 5 pixels from the center
+\item $5<$ {\tt min\_mask\_dist\_pixels} $\leq $ 11 pixels {\em and} {\tt average\_mask\_adjacent\_pixel\_count} is greater than 20\% of the brightest-pixel value.
+\end{enumerate}
+Galaxies with the bad stamp flag in any filter are removed from the final catalogs of both filters.
+We are left with 26,517 good galaxy postage stamps in our final catalog, for both bands.
+
+\subsection{Creating the Noise Map}
+\label{sec:noise_map}
+The drizzling process introduces correlations between neighboring pixels, which artificially reduces the noise level of co-added drizzled images. Care is taken to preserve these correlations when generating noise maps to replace pixel values for nearby objects. 
+ 
+Sixteen empty regions of varying sizes, with dimensions ranging from 70-700 pixels,  at different positions on the image plane, were visually identified. A $50\times50$ covariance matrix for the pixel values was calculated for each empty region (separately in each filter) and their mean computed, as shown in \autoref{fig:covar}.
+Noise maps generated from the mean covariance matrix for each filter are shown in \autoref{fig:noise_map}.
+
+%----------------------------------
+\begin{figure}[h]
+  \centering
+  \subfloat[Covariance map in V (F606W) filter.]{\includegraphics[width=0.4\textwidth]{covar_f814.png}}
+  \hfill
+  \subfloat[Covariance map in I (F814W) filter.]{\includegraphics[width=0.4\textwidth]{covar_f606.png}}
+  \caption{Mean co-variance matrix in the two filters.}
+  \label{fig:covar}
+\end{figure}
+%----------------------------------
+
+%---------------------------------
+\begin{figure}[h]
+\centering\includegraphics[width=1.\linewidth]{noise_map.png}
+\caption{Image of noise map generated from covariance matrices, for I band (left) and V band (right).}
+\label{fig:noise_map}
+\end{figure}
+%----------------------------------
+
+To replace nearby objects with noise, a region that is identical in size and shape to the nearby object in the segmentation map is cut from a random location in the noise map shown in \autoref{fig:noise_map} and used to replace the pixel values for the nearby object in the galaxy postage stamp.  
+
+\subsection{Parametric Fits}
+The COSMOS catalog galaxies were subject to an additional step of fitting for parametric models. This step was not performed here, and the values in the output catalogs corresponding to these parameters were manually set to 999.
```

Comment:
This is not precisely true as stated.  The COSMOS catalog you have referred to so far is the Leauthaud et al (2007) catalog, unless I am missing some other reference.  But that catalog had no parametric fits.  The fits referenced here were produced for the GREAT3 challenge as described in e.g. the GREAT3 handbook.

Reply:
I've now mentioned GREAT3 for the parametric fits, and also added a reference at the beginning that the final products are similar to the great3 datasets.